### PR TITLE
Feature/improve coverage

### PR DESCRIPTION
**Pull request by Le-Caignec**

### CODE REVIEW

File: src/web3mail/sendEmail.ts [hunk 1]
```diff
@@ -28,6 +29,7 @@ export const sendEmail = async ({
 }: IExecConsumer &
   SubgraphConsumer &
   SendEmailParams): Promise<SendEmailResponse> => {
+  let test: IExecOrderbookModule;
```
Comment (SeddikBellamine) [inline src/web3mail/sendEmail.ts:32]: where do you use this test ? i think it's not used

**Review by SeddikBellamine — CHANGES_REQUESTED** [review]

File: src/web3mail/sendEmail.ts (hunk 1 above)
Comment (Le-Caignec) [reply src/web3mail/sendEmail.ts]: Commit: b2fbc599c5514ca8060615a31d8bc16a42a63962

File: src/web3mail/sendEmail.ts [hunk 2]
```diff
@@ -29,7 +29,6 @@ export const sendEmail = async ({
 }: IExecConsumer &
   SubgraphConsumer &
   SendEmailParams): Promise<SendEmailResponse> => {
-  let test: IExecOrderbookModule;
```
Comment (SeddikBellamine) [inline src/web3mail/sendEmail.ts]: remove the import as well of IExecOrderbookModule

Comment (Le-Caignec) [reply src/web3mail/sendEmail.ts]: Commit: 9314d244ae047574102e0a796aba059d701adac7
oups désolé

**Review by SeddikBellamine — APPROVED** [review]
Well done !

**Pull request merged**